model-optimizer/extensions/front/kaldi/set_ports.py

```diff
@@ -28,6 +29,11 @@ def find_and_replace_pattern(self, graph: Graph):
 
             in_ports_count = node.in_ports_count if node.has_valid('in_ports_count') else len(inputs)
             out_ports_count = node.out_ports_count if node.has_valid('out_ports_count') else len(outputs)
+
+            if len(outputs) > out_ports_count > 1:
+                raise Error("Node {} has more children then it should: " +
+                            "should be {} but there is {}".format(node_id, out_ports_count, len(outputs)))
```

Comment:
then -> than.
Please double-check the sentence in terms of grammar correctness.